src/main/java/com/cburch/contracts/BaseWindowFocusListenerContract.java

```diff
@@ -1,3 +1,12 @@
+/*
+ * Logisim-evolution - digital logic design tool and simulator
+ * Copyright by Logisim-evolution developers
```

Comment:
Not all have been replaced, multiple places

Reply:
Last 11 files. Fixed